tests/src/testModels.js

```diff
@@ -803,7 +803,7 @@ describe('Model', function() {
 
         expect(permissionsDefault.variables).toEqual('TEST=yes');
 
-        permissionsDefault.set_property('variables', 'TEST=yes;TE ST=no');
+        permissionsDefault.set_property('variables', 'TEST=yes;TE;ST=no');
```

Comment:
note to future self; find a better way to serialize/serialize this list, in a way that doesn't clash with the actual values...